test/testutil/util.go

```diff
@@ -102,6 +103,8 @@ func ContainsError(err error, message string) bool {
 					return true
 				}
 			}
+		default:
```

Comment:
Out of curiosity, why did you have to add the nolint and why this default block?